organize_data.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
Please move all files with code to separate directory. For example to `src/massive_dataset`.